.typedoc/typedoc-plugin-frontmatter.js

```diff
@@ -0,0 +1,22 @@
+import { ParameterType, PageEvent } from "typedoc";
+
+/**
+ * Adds frontmatter globals to each page.
+ */
+export function load(app) {
```

Comment:
To avoid 2 dependencies (`typedoc-plugin-markdown` + `typedoc-plugin-frontmatter`) that just append a frontmatter blocks, we have this plugin

Reply:
It's API is backward compatible with typedoc-plugin-markdown + typedoc-plugin-frontmatter, so if we feel that we want to use `typedoc-plugin-markdown`, we can switch back to typedoc-plugin-markdown + typedoc-plugin-frontmatter